src/__test__/calculate.test.js

```diff
@@ -0,0 +1,38 @@
+import calculate from '../logic/calculate';
+import obj from '../__mock__/calculate';
+
+describe('Calculate', () => {
+  test('the test is to return an empty object', () => {
+    expect(calculate(obj, 'AC')).toEqual(obj);
+    expect(calculate(obj, 'AC')).toMatchSnapshot();
```

Comment:
- [ ] I think we're using incorrectly the snapshot functionality here. Just checking that calculate returns the expected result, in this case, is enough. Instead, we want to use snapshots to make "copies" of a react component that should rerender the same every time. Let's take a look at this example:

```
import React from 'react';
import renderer from 'react-test-renderer';
import Link from '../Link.react';

it('renders correctly', () => {
  const tree = renderer
    .create(<Link page="http://www.facebook.com">Facebook</Link>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
```
In this case a snapshot of the Link component it's being created and in the future, it should always match the snapshot unless a change to the Link component is made.

- [ ] Please write snapshots tests for your components: Home, Calculator, Quote, etc. [You can read more about how to take snapshots here.](https://jestjs.io/docs/snapshot-testing)